helm/cluster-vsphere/values.schema.json

```diff
@@ -121,8 +121,34 @@
             "title": "Cluster",
             "description": "Helm values for the provider-independent cluster chart.",
             "default": {
+                "internal": {
+                    "advancedConfiguration": {
+                        "controlPlane": {
+                            "apiServer": {
+                                "extraArgs": {
+                                    "requestheader-allowed-names": "front-proxy-client"
```

Comment:
we're not entirely sure why this is in the vsphere chart - it may be possible to drop it but as we're not sure i've elected to keep it for now.